test/lib/userFactory.test.js

```diff
@@ -5,13 +5,15 @@ const mockery = require('mockery');
 const sinon = require('sinon');
 
 sinon.assert.expose(assert, { prefix: '' });
+require('sinon-as-promised');
```

Comment:
Sinon version 2 does, we're still using 1.17.4. There are a lot of places using sinon-as-promised that could be converted to just using sinon, if we want to switch to sinon 2 I feel like that should be a separate PR.